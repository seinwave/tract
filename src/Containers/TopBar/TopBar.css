.topbar-logo-image {
    height: 40px;
}
.topbar-wrapper {
    background-color: #fff
}

.navbar{
    border-bottom: .5px solid #e4e9f0;
}

.avatar-dropdown-button button {
    background-color: transparent;
    border-radius: 15px;
    border: none;
    color: black;
}

.avatar-dropdown-button button:hover{
    background-color: #d3d3d3;
    border-radius: 15px;
    border: none;
    color: black;
}

.topbar-avatar-image {
    height: 40px;
}