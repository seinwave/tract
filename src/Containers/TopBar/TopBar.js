import React from "react";
import Dropdown from "react-bootstrap/Dropdown";
import "./TopBar.css";

function TopBar() {
  return (
    <div className="topbar">
      <nav className="navbar navbar-expand-lg navbar-light">
        <img
          src="https://1000logos.net/wp-content/uploads/2017/11/Chipotle-Logo.png"
          alt="tract-logo"
          className="topbar-logo-image"
        />
        <a className="navbar-brand" href="www.mattseidholz.com/">
          Tract
        </a>
        <div
          className="collapse navbar-collapse"
          id="navbarSupportedContent"
        ></div>
        <Dropdown className="avatar-dropdown-button">
          <Dropdown.Toggle>
            <img
              src="https://cdn.iconscout.com/icon/free/png-256/avatar-375-456327.png"
              alt="Avatarimage"
              className="topbar-avatar-image"
            />{" "}
            Hi, USER
          </Dropdown.Toggle>
          <Dropdown.Menu>
            <Dropdown.Item href="#/action-1">Action</Dropdown.Item>
            <Dropdown.Item href="#/action-2">Another action</Dropdown.Item>
            <Dropdown.Item href="#/action-3">Something else</Dropdown.Item>
            <Dropdown.Divider />
            <Dropdown.Item href="#/action-3">Sign out</Dropdown.Item>
          </Dropdown.Menu>
        </Dropdown>
      </nav>
    </div>
  );
}

export default TopBar;
