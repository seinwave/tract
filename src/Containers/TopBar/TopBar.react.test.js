import React from "react";
import TopBar from "./TopBar";
import renderer from "react-test-renderer";

it("renders as expected", () => {
  const tree = renderer.create(<TopBar />).toJSON();
  expect(tree).toMatchSnapshot();
});
