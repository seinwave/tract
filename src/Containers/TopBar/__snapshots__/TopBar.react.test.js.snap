// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as expected 1`] = `
<div
  className="topbar"
>
  <nav
    className="navbar navbar-expand-lg navbar-light"
  >
    <img
      alt="tract-logo"
      className="topbar-logo-image"
      src="https://1000logos.net/wp-content/uploads/2017/11/Chipotle-Logo.png"
    />
    <a
      className="navbar-brand"
      href="www.mattseidholz.com/"
    >
      Tract
    </a>
    <div
      className="collapse navbar-collapse"
      id="navbarSupportedContent"
    />
    <div
      className="avatar-dropdown-button dropdown"
      onKeyDown={[Function]}
    >
      <button
        aria-expanded={false}
        aria-haspopup={true}
        className="dropdown-toggle btn btn-primary"
        disabled={false}
        onClick={[Function]}
        type="button"
      >
        <img
          alt="Avatarimage"
          className="topbar-avatar-image"
          src="https://cdn.iconscout.com/icon/free/png-256/avatar-375-456327.png"
        />
         
        Hi, USER
      </button>
    </div>
  </nav>
</div>
`;
